### ISSUE TITLE
Importing new version of the data model using Excel wipes out all classifiers previously assigned to a data model

### ISSUE THREAD

**Issue by jelenadreskai** [MauroDataMapper/mdm-core#205]
**Describe the bug**
Importing new version of the data model using Excel Importer wipes out all classifiers previously assigned to a data model.
Our end users maintain data models using Excel.

**To Reproduce**
1. Create a data model and assign a few classifiers to it.
2. Export data model
3. Make changes to the data model description (or any other changes) in the Excel
4. Re-import data model version using Excel Importer

Result:
New version of the data model does not have classifiers that were previously attached to it.

**Expected behavior**
Classifiers should not be removed from the data model, since they were not changed in the Excel.

Note: Classifiers do not exist in the Excel Importer at all. The other option is to start including classifiers in the Excel.

**Additional context**
* MDM Core 4.10.0
* MDM UI: 6.6.0

**Comment by olliefreeman** [comment]
We should add this to the functionality developed as part of MC-9536. No need for addtl parameters, just make it part of that propagation

**Comment by olliefreeman** [comment]
@aaronforshaw while you do this can you update/override the methods inside the datamodel plugin to propagate the Summary Metadata, its why mc-9536 is still open as we missed it in the release

**Comment by aaronforshaw** [comment]
Classifiers are already propagated, but I guess (not certain) this work was released in B4.11.0_F6.7.0.

@jelenadreskai I ran your scenario on the latest code, but with the additional note in step 4 as follows:

4. Re-import data model version using Excel Importer, checking the "Import as New Branch Model Version" and "Propagate From Previous Version" checkboxes

and found that classifiers were correctly copied to my newly imported model.
